### ISSUE TITLE
Support to Build rss/atom feeds

### ISSUE THREAD

**Issue by idlip** [tonyaldon/one.el#3]
Hi @tonyaldon!

Its so nice to see ssg with pure org-mode and elisp. I just tried it now, and I can feel how simple and friendly to org-mode it is.

I have to still dig and try out more possibility to get desired themes. 

Just one required feature would be support to generate rss/atom feeds.
This would be easy, even I will give a try of writing elisp that generates it.

Just adding it here, not an issue as such, just a feature request.

**Comment by tonyaldon** [comment]
Thanks @idlip for giving a try to `one.el`.

> Just one required feature would be support to generate rss/atom feeds.
> This would be easy, even I will give a try of writing elisp that
> generates it.

I have not planned to add functions to build rss/atom feeds.  I think this can be done on a per project basis.  But I leave that issue open in case I change my mind in the future.

I just published [tonyaldon/minibuffer.tonyaldon.com](https://github.com/tonyaldon/minibuffer.tonyaldon.com) repository which contains the code/content of [minibuffer.tonyaldon.com](https://minibuffer.tonyaldon.com).  You can find in its `onerc.el` file an example of how to produce a `feed.xml` file using `one-hook`.  Specifically, check the function `mb-feed`.

I hope this is helpful.

**Comment by idlip** [comment]
Yes, that helps it. Jack looks like like it makes writing html in elisp very simple.

Thank you for sharing and your work on both the package! @tonyaldon 

I'm sure with emacs conf, many people would be trying this and appreciate the design.
Maybe in some stream daviwil (system crafters) can showcase live!

**Comment by tonyaldon** [comment]
> Yes, that helps it.

Cool!

> Maybe in some stream daviwil (system crafters) can showcase live!

Would be great!

**Comment by idlip** [comment]
Also @tonyaldon one issue I stumbled on, May I know where is `(org-font-lock-ensure)` is defined?

I get `progn: Symbol’s function definition is void: org-font-lock-ensure`

**Comment by idlip** [comment]
Another doubt, sorry that I'm asking it here only (did not wanted to open another issue)

Normal org export to html file inherits the src block's syntax highlight from emacs theme itself, so is there a way or option to do it that way for `one-build` too?

I saw there are `one-hl-*` classes. But since emacs theme itself gives us syntax highlight, it would be easy for newcomers to get started with good blogs with syntax highlight.

**Comment by tonyaldon** [comment]
I think the information you are looking for is in "one-ox | src-block":

https://one.tonyaldon.com/docs/one-ox-src-block/

Let me know if this answer to your question.

**Comment by tonyaldon** [comment]
> I get progn: Symbol’s function definition is void:
> org-font-lock-ensure while one-build

The function `org-font-lock-ensure` has been remove in org-mode 9.6.6.

So I removed it in 597dce0.

Try to use `one.el` v1.1.

**Comment by idlip** [comment]
> https://one.tonyaldon.com/docs/one-ox-src-block
Oh yes, it is what I needed to read. I rushed with excitement that I have not read the docs properly. lol

> Try to use one.el v1.1.

Funny that I had cloned this repo to try when you posted on reddit itself, `git pull` and yes sir, its all good now!

Thank you for kind response and clarification!

**Comment by tonyaldon** [comment]
You're welcome
